r2/r2/templates/base.html

```diff
@@ -30,7 +30,8 @@
   <head>
     <title>${self.Title()}</title>
 
-    <meta name="viewport" content="initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0"/>
+<!-- `meta name="viewport"` set for mobile browsers. See https://github.com/tog22/eaforum/issues/7 & https://github.com/tog22/eaforum/issues/28 -->
```

Comment:
It's much better to put this sort of information in the commit message. Comments in code almost invariably end up becoming outdated and irrelevant. Will merge anyway but please note in future.